src/Magento/FunctionalTestingFramework/Console/BaseGenerateCommand.php

```diff
@@ -28,6 +29,23 @@ protected function configure()
             'r',
             InputOption::VALUE_NONE,
             'remove previous generated suites and tests'
+        )->addOption(
+            "force",
+            'f',
+            InputOption::VALUE_NONE,
+            'force generation of tests regardless of Magento Instance Configuration'
+        )->addOption(
+            "allowSkipped",
+            'a',
+            InputOption::VALUE_NONE,
+            'Allows MFTF to generate and run skipped tests.'
+        )->addOption(
+            'debug',
+            'd',
+            InputOption::VALUE_OPTIONAL,
+            'Run extra validation when running tests. Use option \'none\' to turn off debugging -- 
+             added for backward compatibility, will be removed in the next MAJOR release',
+            MftfApplicationConfig::LEVEL_DEFAULT
```

Comment:
Similar like previous comment. mention both "generate" and "run".

Reply:
Will do 👍